modules/core/src/main/scala/fs2/kafka/KafkaConsumer.scala

```diff
@@ -333,6 +333,15 @@ object KafkaConsumer {
         }
       }
 
+      override def signedPartitionedStream: Stream[F, (TopicPartition, Stream[F, CommittableConsumerRecord[F, K, V]])] = {
+        partitionsMapStream.flatMap { partitionsMap =>
+          Stream.emits(partitionsMap.toVector.map {
+            case (topicPartition, partitionStream) =>
+              topicPartition -> partitionStream
+          })
```

Comment:
Can't this just be the following?
```suggestion
          Stream.emits(partitionsMap.toVector)
```

Reply:
I know I can but IMO still missing it in API